ossindex/ossindex.go

```diff
@@ -86,8 +87,20 @@ func openDb(dbDir string) (db *badger.DB, err error) {
 	return
 }
 
-// AuditPackages will given a list of Package URLs, run an OSS Index audit
+// AuditPackages will given a list of Package URLs, run an OSS Index audit.
+//
+// Deprecated: AuditPackages is old and being maintained for upstream compatibility at the moment.
```

Comment:
Maybe open an issue so that we can clean this up before we cut a 1.0.0 official release??

Reply:
YES ZENDER SIR! 🖖